docs/root/configuration/http_filters/ext_authz_filter.rst

```diff
@@ -0,0 +1,51 @@
+.. _config_http_filters_ext_authz:
+
+External Authorization
+======================
+* External authorization service overview :ref:`architecture overview <arch_overview_ext_authz>`
+* :ref:`HTTP filter v2 API reference <envoy_api_msg_config.filter.http.ext_authz.v2alpha.ExtAuthz>`
+
+The external authorization HTTP filter calls an external authorization service to check if the incoming HTTP request is authorized or not.
```

Comment:
Would be better to call `authorization gRPC filter` instead, so it doesn't get confused with the upcoming changes `authorization raw HTTP filter`? WDYT?

Reply:
I've reworded this.